packages/modeling/src/operations/transforms/mirror.js

```diff
@@ -27,26 +27,41 @@ const mirror = (options, ...objects) => {
   if (objects.length === 0) throw new Error('wrong number of arguments')
 
   const planeOfMirror = plane.fromNormalAndPoint(normal, origin)
```

Comment:
good & clean updates ! nice !
One question about mirror itself that I had not thought about before though : can you not achieve the same thing as giving a specific origin with a translated version of the object you want to mirror ? aka why do we need both normal & origin ?

Reply:
Normal and origin (point) are required to create an arbritary plane, i.e. plane.fromNormalAndPoint(). And, the plane is used to create a matrix, i.e. mat4.mirrorByPlane()

Of course, origin is optional, with the default being 0,0,0.

Also, V1 API only allowed mirror about 0,0,0.

Reply:
Ok thanks for clarifying :) 
But my question still stands, can you not achieve the same effect by using a translation and a mirror about 0,0,0 ?

Reply:
Sure. Sounds possible.

Reply:
in that case why not have the simpler api ? ie just having to provide the normal ?

Reply:
Let me try this... I’ll try some different scenarios of mirror using mirror about 0,0'0, and translate.

Reply:
Well... It turns out that mirror (normal, origin) is not the same as translate(origin) + mirror(normal)

```
const { color, geometry, primitives, transforms } = require('@jscad/modeling')

const main = (params) => {
  let shape1 = primitives.cuboid({size: [10, 15, 5]})
  let shapeA = transforms.center({center: [-20,30,0]}, shape1)
  let shapeB = transforms.center({center: [-20,-30,0]}, shape1)
  let shapeC = transforms.center({center: [20,30,0]}, shape1)
  let shapeD = transforms.center({center: [20,-30,0]}, shape1)
  color.color([1, 0, 0, 1], shapeA, shapeB, shapeC, shapeD)
  let shapes = [shapeA, shapeB, shapeC, shapeD]

  let mirrorsA = transforms.mirror({normal: [15,15,15], origin: [15,15,15]}, shapes)
  geometry.geom3.toPolygons(mirrorsA[0])
  geometry.geom3.toPolygons(mirrorsA[1])
  geometry.geom3.toPolygons(mirrorsA[2])
  geometry.geom3.toPolygons(mirrorsA[3])
  color.color([0, 1, 0, 1], mirrorsA)

  let mirrorsB = transforms.translate([15,15,15], shapes)
  mirrorsB = transforms.mirror({normal: [15,15,15], origin: [0,0,0]}, mirrorsB)
  geometry.geom3.toPolygons(mirrorsB[0])
  geometry.geom3.toPolygons(mirrorsB[1])
  geometry.geom3.toPolygons(mirrorsB[2])
  geometry.geom3.toPolygons(mirrorsB[3])
  color.color([0, 0, 1, 1], mirrorsB)

  return [...shapes, ...mirrorsA, ...mirrorsB]
}

module.exports = { main }
```

P.S. Comment out one of those toPolygons(), and the shape will rotate as the camera moves.